modules/mobile/config/routes.rb

```diff
@@ -33,7 +33,6 @@
     get '/health/locations/:id', to: 'locations#show'
     scope :health do
       get 'labs-and-tests', to: 'labs_and_tests#index'
-      resources :labs_and_tests, only: %i[index], defaults: { format: :json }
       resources :observations, only: %i[show], defaults: { format: :json }
```

Comment:
Why do we need to specify format here? I'm not really sure why we're so inconsistent about that in our routes file. I think all of our routes are json, so we could probably just set the default for all of them.

Reply:
I removed it, agreed we should probably just set the default universally